README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Hello everyone, I'm **Dora Gonzalez** and it is nice to virtually meet you all! ##
+
+<img src= "https://github.com/Dora-MG/hw01/blob/master/space%20buns%20.jpg?raw=true" width="350"> 
+
+I am a **MPP** student at the *Harris School of Public Policy*. 
+
+Feel free to stay connected via [LinkedIn](https://www.linkedin.com/in/doragonzalez/). 
+
+Some of my favorite hobbies include: 
```

Comment:
I would be careful with **spacing** here. Generally, we like to see **as little unnecessary space as possible** to make sure the code is concise and readable. This will be more important for future assignments when we start to look more closely at coding style

Reply:
Can you clarify spacing where? After [LinkedIn]? or after the link and "some of my favorite hobbies include:" ?

Reply:
@djanvier Can you also explain how to embed an image in r markdown if we don't have it in our github repository?  For example, if I just want to use an image saved on my desktop or an imagine I got online?